functorch/_src/eager_transforms.py

```diff
@@ -594,7 +588,7 @@ def noop():
     yield
 
 
-def assert_flat_tuple_of_tensors(elts, api, argname):
+def assert_flat_tuple_of_tensors(elts: Tuple[torch.Tensor, ...], api: str, argname: str) -> None:
```

Comment:
elts should be typed as Any, right? It's not guaranteed that we'll be pass a Tuple of Tensors to this function. mypy (if it ran on this file) should end up complaining that we are passing things that may be statically known to not be tuples to this function.

Reply:
Actually, I do not see in the codebase where this method is used. 
IMO the purpose of typing hints is to show expected type. If somewhere in the codebase we used it on an unacceptable data (which would raise an error) and mypy would complain about that, it is good for us to see a bad usage of the method statically and not on the runtime. I assume that raising `RuntimeError` is not desired behaviour.